tracer/test/snapshots/deduplication.deduplicated.verified.txt

```diff
@@ -0,0 +1,36 @@
+﻿[
+  {
+    TraceId: Id_1,
```

Comment:
I'm not good at that but should those spans have a specific `origin` if they don't had one already? Isn't this how we identify products? cc @lucaspimentel to correct me

Reply:
That would make sense. I have seen that in app_sec it is addded also.

Reply:
After discussing the topic with the IAST team, the origin tag will be added. Thanks for the suggestion!